docs/developers/sdk/analytics.rst

```diff
@@ -33,6 +33,10 @@ Matomo integration relies on ``window._paq`` being available.
 
 Google Analytics uses the ``window.ga`` global variable.
 
+**Piwik PRO**
+
+Piwik PRO uses the ``_paq`` global variable.
```

Comment:
I'm not seeing the PR in the SDK to support Piwik PRO? https://github.com/open-formulieren/open-forms-sdk

Reply:
Seems to be covered, but let's at least add a comment that this also supports Piwik PRO: https://github.com/open-formulieren/open-forms-sdk/blob/main/src/hooks/usePageViews.js#L19